Scripts/Runtime/Entities/Util/ChunkUtil.cs

```diff
@@ -0,0 +1,26 @@
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Mathematics;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    /// <summary>
+    /// Helper class for dealing with optimal numbers of elements in a chunk
+    /// </summary>
+    public static class ChunkUtil
+    {
+        //See Chunk.kChunkSize - Can't access so we redefine here
+        private const int CHUNK_SIZE = 16 * 1024;
+        
+        /// <summary>
+        /// Gets the maximum number of elements that would fit into a chunk of memory.
+        /// At minimum it will return 1 if the element is larger than a chunk.
```

Comment:
I wonder if we should emit a warning if this is the case.
I guess you still allocate the right size if `type > CHUNK_SIZE` so it's not going to cause problems.

Reply:
Yeah, it won't break and I feel like you're probably going to know if you have a struct that is that massive?

Reply:
That's fair. Would be annoying if you, for some reason, had to work with a massive struct.